tests/std/tests/P0218R1_filesystem/test.cpp

```diff
@@ -3570,6 +3570,7 @@ void test_temp_directory_path() {
 void test_create_directory() {
     const test_temp_directory tempDir("create_directory"sv);
     const path p = tempDir.directoryPath / L"__std.c++17.filesystem.create_directory"sv;
+    const path emptyPath{};

```

Comment:
If we don't need an lvalue for any reason, I think `path{}` is even more clearly and readably an empty path than is `emptyPath`. Why not simply use `path{}`?